schema.lua

```diff
@@ -59,10 +36,16 @@ return {
     maximum_latency_msec = {type = "integer", required = true, func = check_maximum, default = 1000},
     request_percentage = {type = "integer", required = true, func = check_percentage, default = 50},
     add_header = {type = "boolean", default = true},
-    
+
   },
   self_check = function(schema, plugin_t, dao, is_updating)
-    -- perform any custom verification
+    local maximum = tonumber(plugin_t.maximum_latency_msec) or -1
```

Comment:
an example of how the self_check function works, where values can be checked in relation to other values.
Note that this is gone in the 1.0 version  of Kong, and hence also in the next Enterprise release 0.35.